IFComp/lib/IFComp/Controller/Play.pm

```diff
@@ -98,17 +98,15 @@ sub transcribe : Chained('fetch_entry') : Args(0) {
 
     my $entry = $c->stash->{entry};
 
-    my $data_ref = $c->req->body_data->{data};
-    unless ( ref $data_ref ) {
-        $data_ref = from_json($data_ref);
-    }
-
+    # Quixe (natively) and Parchment (via if-recorder) report transcription
```

Comment:
I wouldn't mind seeing the block of code moved out to a separate $self->_normalize_transcript_data method, and letting that method handle figuring out which of the _normalize_*_transcript_data() methods to call.